packages/core/src/server.ts

```diff
@@ -79,6 +79,10 @@ export class ServerConfig extends Config {
     @ConfigParam("boolean")
     verifyEmailOnSignup = true;
 
+    /** Whether or not to disable creating an account via the signup form */
+    @ConfigParam("boolean")
+    disableSignup = false;
```

Comment:
Doesn't this prevent signup altogether? That's not what we want as it would prevent onboarding even for provisioned members (even provisioned members need to complete the signup process). What we want instead is a setting to change the default provisioning status, i.e. the provisioning status for users for which no status has been set explicitly. Also, this setting should be on the provisioner, not the server itself.

Reply:
I'll make that change myself later today. You'll see what I mean.

Reply:
Gotcha, and yes, it prevents signup. This was meant to be a setting that wasn't directly related to provisioning, just something that someone might want to do on their server, for example, and helped if you just wanted directory provisioning, for example.

Reply:
I guess there was a misunderstanding on how provisioning works. The provisioner simply "whitelists" a given email address, but doesn't actually create the account. So the user still has to go through the signup flow, even if their provisioning status has been set to `active`.

Reply:
Ohh, indeed. This makes more sense, then. I guess this isn't necessary at all, then.